components/table/test/table.vdiff.js

```diff
@@ -0,0 +1,1229 @@
+import '../../inputs/input-number.js';
+import '../../inputs/input-text.js';
+import '../demo/table-test.js';
+import '../table-col-sort-button.js';
+import { defineCE, expect, fixture, focusElem, html, nextFrame } from '@brightspace-ui/testing';
+import { LitElement } from 'lit';
+import { tableStyles } from '../table-wrapper.js';
+
+const colSortButtonFixture = html`
+	<table class="d2l-table">
+		<thead>
+			<tr>
+				<th><d2l-table-col-sort-button>Ascending</d2l-table-col-sort-button></th>
+				<th><d2l-table-col-sort-button desc>Descending</d2l-table-col-sort-button></th>
+				<th><d2l-table-col-sort-button nosort>No Sort</d2l-table-col-sort-button></th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td>Cell 1-A</td>
+				<td>Cell 1-B</td>
+				<td>Cell 1-C</td>
+			</tr>
+		</tbody>
+	</table>
+`;
+
+describe('d2l-table', () => {
+
+	['ltr', 'rtl'].forEach((dir) => {
+		describe(dir, () => {
+			['default', 'light'].forEach((type) => {
+
+				async function createFixture(markup, opts) {
+					const rtl = (dir === 'rtl');
+					const tag = defineCE(
+						class extends LitElement {
+							static get styles() {
+								return [tableStyles];
+							}
+							render() {
+								const wrapper = html`
+									<d2l-table-wrapper
+										?no-column-border="${opts?.noColumnBorder}"
+										?sticky-headers="${opts?.stickyHeaders}"
+										style="--d2l-input-position: static;"
+										type="${type}">${markup}</d2l-table-wrapper>`;
+								if (!opts?.bottomMargin) {
+									return wrapper;
+								}
+								return html`<div style="margin-bottom: 1000px;">${wrapper}</div>`;
```

Comment:
This is so that there's room at the bottom to scroll. I'm going to play around with this and look into applying right/left margin as well as maybe that's the source of some of the scrolling diff issues.